src/api.rs

```diff
@@ -1830,6 +1830,7 @@ pub struct ReleaseInfo {
     pub last_event: Option<DateTime<Utc>>,
     #[serde(rename = "newGroups")]
     pub new_groups: u64,
+    pub projects: Vec<ProjectSlugAndName>,
```

Comment:
Could you please add `#[serde(default)]` here, to be safe? IIRC, old versions of this endpoint did not return a list of projects.